lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
```

Comment:
5% is a lot, but `try..catch` function, (like from <a href="https://github.com/petkaantonov/bluebird/wiki/Optimization-killers">here</a>)  probably wouldn't get us anywhere, it worth a try though